wordpress-mobile/MediaPicker-iOS
# Refactor date formatting

This PR just does some refactor around date/time formatting code to avoid code duplication and make things more consistent for our users.

To test:
 - Run the demo app
 - See if the time on videos shows correctly on thumbs cells and when previewing video full screen.
